src/main/java/org/bigredbands/mb/views/MainView.java

```diff
@@ -76,9 +76,11 @@ private void initialize() {
 
     /**
      * Create the project view and display it
+     * 
+     * @param loaded - true if the user is loading the project, false if this is a new project
```

Comment:
This is a fairly subtle point (so apologies if I don't explain it well), but I find it better to name variables based on "what they do" rather than "when/how they expect the caller to set them." The general reason behind that is that software should [avoid embedding "business logic"](https://medium.com/expedia-group-tech/six-software-anti-patterns-that-kill-your-delivery-4fed09cc1a2b#caf5) where it isn't explicitly needed.

In this case, the name `loaded` indicates a _user_ behavior - that they've loaded a `.pnd` file rather than creating from "New". That inextricably links downstream behavior (in this case, not showing the Song Constants wizard) with the user action (opening a `.pnd` file). That decision has two consequences: 

1. Objects all through the call stack are operating based on the user's behavior, so they all need to be designed with the overall application behavior in mind. If we ever wanted to change that behavior (e.g. open the Song Constants for new drill files _and_ when Song Name is empty), we'd need to modify that boolean all the way through the call stack. 
2. If `loaded` refers to the overarching application state, different components can start tacking on additional behavior (e.g., maybe `ProjectView` shows up in a different color when loaded from a `.pnd` file). That custom behavior is then scattered across the codebase, making it very difficult to track what exactly happens from a user-facing perspective when loading from a `.pnd` file.

With that in mind, a name I'd recommend would be something like `boolean showWizard`. It's narrowly focused on exactly what we want `ProjectView` to do - in this case, either show the Song Constants wizard or not - without giving `ProjectView` any indication as to _why_ it's doing that. That contains all the "business logic" in `MainController`, where it's much easier to see the connection between `loadProject()` -> `showWizard = false` and `createEmptyProject()` -> `showWizard = true`.